### PR TITLE
Routing, websocket fix, deprecate old connections

### PR DESCRIPTION
**Pull request by TelegramSam**

### CODE REVIEW

File: src/renderer/components/Routing/index.vue [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+}
+.basicmessage-Received {
+  background-color: lightblue;
+  margin-left: 4em;
+  margin-bottom: 1em;
+  padding: 1em;
+  text-align: right;
+  border: 1px solid lightgrey;
+  border-radius: 4px;
+}
```
Comment (dbluhm) [inline src/renderer/components/Routing/index.vue:91]: Seems to be unused

File: src/renderer/components/Mediator/index.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+}
+.basicmessage-Received {
+  background-color: lightblue;
+  margin-left: 4em;
+  margin-bottom: 1em;
+  padding: 1em;
+  text-align: right;
+  border: 1px solid lightgrey;
+  border-radius: 4px;
+}
```
Comment (dbluhm) [inline src/renderer/components/Mediator/index.vue:49]: Seems to be unused

File: src/renderer/components/MessageHistory/index.vue [hunk 3]
```diff
@@ -43,12 +46,16 @@ export const shared = {
...
+  inject: ['get_connection'],
   mixins: [
     message_bus(),
     share({
       use: ['message_history']
     })
   ],
+  computed: {
+
+  },
```
Comment (dbluhm) [inline src/renderer/components/MessageHistory/index.vue:58]: Empty computed, probably best to remove for the sake of cleanliness

**Review by dbluhm — COMMENTED** [review]
With #167 merged, it would be helpful to isolate these changes from its change set. Docs for the new messages would also be appreciated.

**Comment by TelegramSam** [comment]
Removed unneeded elements.

As far as isolated changes: timing and a moving target made it hard to complete an isolated changeset. How much is it worth splitting up the updates?

**Review by dbluhm — APPROVED** [review]
Comments addressed; looks good to me

**Pull request merged**